{{ cookiecutter.app_name }}/pyproject.toml

```diff
@@ -46,19 +48,127 @@ requires = [
 requires = [
 {%- if cookiecutter.gui_framework == "Toga" %}
     "toga-gtk~=0.3.0",
-{% endif -%}
+{%- endif %}
+]
+
+[tool.briefcase.app.{{ cookiecutter.app_name }}.linux.system.debian]
+system_requires = [
+{%- if cookiecutter.gui_framework == "Toga" %}
+    # Needed to compile pycairo wheel
+    'libcairo2-dev',
+    # Needed to compile PyGObject wheel
+    'libgirepository1.0-dev',
+{%- elif cookiecutter.gui_framework == "PursuedPyBear" %}
+# ?? FIXME
+{%- endif %}
+]
+
+system_runtime_requires = [
+{%- if cookiecutter.gui_framework == "Toga" %}
+    # Needed to provide GTK
+    "libgtk-3-0",
+    # Needed to provide GI bindings to GTK
+    "libgirepository-1.0-1",
+    "gir1.2-gtk-3.0",
+    # Needed to provide WebKit2 at runtime
+    # "libwebkit2gtk-4.0-37",
+    # "gir1.2-webkit2-4.0",
+{%- elif cookiecutter.gui_framework == "PySide2" or cookiecutter.gui_framework == "PySide6" %}
+    # Derived from https://doc.qt.io/qt-6/linux-requirements.html
+    "libxrender1",
+    "libxcb-render0",
+    "libxcb-render-util0",
+    "libxcb-shape0",
+    "libxcb-randr0",
+    "libxcb-xfixes0",
+    "libxcb-xkb1",
+    "libxcb-sync1",
+    "libxcb-shm0",
+    "libxcb-icccm4",
+    "libxcb-keysyms1",
+    "libxcb-image0",
+    "libxcb-util1",
+    "libxkbcommon0",
+    "libxkbcommon-x11-0",
+    "libfontconfig1",
+    "libfreetype6",
+    "libxext6",
+    "libx11-6",
+    "libxcb1",
+    "libx11-xcb1",
+    "libsm6",
+    "libice6",
+    "libglib2.0-0",
+    "libgl1",
+    "libegl1-mesa",
+    "libdbus-1-3",
+    "libgssapi-krb5-2",
+{%- elif cookiecutter.gui_framework == "PursuedPyBear" %}
+# ?? FIXME
+{%- endif %}
+]
+
+[tool.briefcase.app.{{ cookiecutter.app_name }}.linux.system.rhel]
+system_requires = [
+{%- if cookiecutter.gui_framework == "Toga" %}
+    # Needed to compile pycairo wheel
+    'cairo-gobject-devel',
+    # Needed to compile PyGObject wheel
+    'gobject-introspection-devel',
+{%- elif cookiecutter.gui_framework == "PursuedPyBear" %}
+# ?? FIXME
+{%- endif %}
+]
+
+system_runtime_requires = [
+{%- if cookiecutter.gui_framework == "Toga" %}
+    # Needed to provide GTK
+    "gtk3",
+    # Needed to provide WebKit2 at runtime
+    # "webkit2gtk3",
```

Comment:
I needed `gobject-introspection` at runtime for the `toga` HelloWorld app to run.